docs/aws-iam-setup.md

```diff
@@ -0,0 +1,120 @@
+# Setting up AWS IAM authentication
+
+This document provides guidelines how to configure Percona Server for MongoDB to use AWS IAM authentication. The use of this authentication method enables you to natively integrate Percona Server for MongoDB with AWS services, increase security of your infrastructure by setting up password-less authentication and offload your DBAs from managing different sets of secrets. To learn more, see [AWS IAM authentication](aws-iam.md)
+
+To configure AWS IAM authentication means to set up your AWS environment and configure Percona Server for MongoDB. The AWS environment setup is out of scope of this document. Consult the AWS documentation to perform the following setup steps: 
+
+1.	[Configure the AWS service to work with IAM and support temporary credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_aws-services-that-work-with-iam.html)
+2.	For user authentication: 
+      
+      * [Create the IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) and copy its ARN (Amazon Resource Name)
+
+    For role authentication:
+
+      * [Create the IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html)
+      * Attach the IAM role to the AWS service.
+      * Copy the ARN of the IAM role. 
+
+## Configure Percona Server for MongoDB
+
+The steps are the following:
+
+1. Add records for the IAM user/role in the `$external` database
+2. Enable external authentication
```

Comment:
Not sure what does it mean to "enable external authentication". Indeed we need to enable authentication but after that we select external authentication by specifying user's authentication database in connection parameters.

Reply:
After reading details below I understand the idea that we actually need to do two things:
- enable authentication
- enable MONGODB-AWS mechanism
But the "external authentication" term also includes LDAP and Kerberos authentication so I suggest to not use it here.